build.gradle.kts

```diff
@@ -102,7 +104,9 @@ dependencies {
     include(libs.hypixelapi)
     include(libs.skyblockapi)
     include(libs.rconfig)
-    include(libs.rconfigkt)
+    include(libs.rconfigkt) {
+        isTransitive = false
```

Comment:
is this even needed since it's just the include and not the compiletimeClasspath